docs/docs/aztec/aztec/concepts/index.md

```diff
@@ -51,15 +51,15 @@ Aztec allows private communications with Ethereum - ie no-one knows where the tr
 
 This is achieved through portals - these are smart contracts deployed on an EVM that are related to the Ethereum smart contract you want to interact with.
 
-Learn more about portals [here](./communication/cross_chain_calls.md).
+Learn more about portals [here](/aztec/aztec/concepts/smart_contracts/communication/cross_chain_calls).
 
 ## Circuits
 
 Aztec operates on three types of circuits:
 
-- [Private kernel circuits](./circuits/kernels/private_kernel.md), which are executed by the user on their own device and prove correct execution of a function
-- [Public kernel circuits](./circuits/kernels/public_kernel.md), which are executed by the [sequencer](./nodes_clients/sequencer/main.md) and ensure the stack trace of transactions adheres to function execution rules
-- [Rollup circuits](./circuits/rollup_circuits/main.md), which bundle all of the Aztec transactions into a proof that can be efficiently verified on Ethereum
+- [Private kernel circuits](/aztec/aztec/concepts/circuits/kernels/private_kernel.md), which are executed by the user on their own device and prove correct execution of a function
+- [Public kernel circuits](./circuits/kernels/public_kernel.md), which are executed by the [sequencer](/aztec/aztec/concepts/nodes_clients/sequencer) and ensure the stack trace of transactions adheres to function execution rules
+- [Rollup circuits](/aztec/aztec/concepts/circuits), which bundle all of the Aztec transactions into a proof that can be efficiently verified on Ethereum
```

Comment:
Advantages of referring to docs as relative files mentioned [here](https://docusaurus.io/docs/markdown-features/links). 
Best practice of relative files for versioned docs [here](https://docusaurus.io/docs/versioning#link-docs-by-file-paths), is there a way to test absolute urls with different versions?
Namely absolute urls (no extension) which we did not have in `master` branch.

If extension removal and path absolutification are their own atomic commits, it'll be easy to separate them to a follow up PR where the effects of this on versions can be tested and have the restructure land preserving path types as they were.

Reply:
yeah they're not in their own commits, but I can write a script to recursively look into each folder and rewrite paths